packages/api/src/ontology/Definition.ts

```diff
@@ -27,26 +31,51 @@ export type ObjectInfoFrom<
 export type ObjectTypesFrom<O extends OntologyDefinition<string>> =
   keyof O["objects"];
 
-export type PropertyKeysFrom<
+export type ObjectPropertyKeysFrom<
   O extends OntologyDefinition<any>,
   K extends ObjectTypesFrom<O>,
 > = keyof ObjectInfoFrom<O, K>["properties"];
 
-export type PropertyDefinitionsFrom<
+export type ObjectPropertyDefinitionsFrom<
   O extends OntologyDefinition<any>,
   K extends ObjectTypesFrom<O>,
 > = ObjectInfoFrom<O, K>["properties"];
 
-export type PropertyDefinitionFrom<
+export type ObjectPropertyDefinitionFrom<
   O extends OntologyDefinition<any>,
   K extends ObjectTypesFrom<O>,
-  P extends PropertyKeysFrom<O, K>,
-> = PropertyDefinitionsFrom<O, K>[P];
+  P extends ObjectPropertyKeysFrom<O, K>,
+> = ObjectPropertyDefinitionsFrom<O, K>[P];
+
+export type InterfaceInfoFrom<
+  O extends OntologyDefinition<any>,
+  K extends InterfaceNamesFrom<O>,
```

Comment:
Shall we start using `I` instead of `K` which would normally be the objectTypeApi?